ui/components/package.json

```diff
@@ -41,8 +41,8 @@
     "@bentley/imodeljs-quantity": "workspace:^3.0.0-dev.22",
     "@bentley/ui-abstract": "workspace:^3.0.0-dev.22",
     "@bentley/ui-core": "workspace:^3.0.0-dev.22",
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0"
+    "react": "^17.0.0",
```

Comment:
should we set a peerdep to be ^16.8 || ^17 for a few weeks just so we dont break anyone who is pulling the 3.0.0-dev.x versions and give them some time to update, and a month or so before we officially release we bump this to ^17.

Only asking this because this change is fine in this monorepo, but for the itwin-viewer, we have dependencies outside our monorepo on vcr (tree-widget-react & property-grid-react) and that means we need to go an update those, and create new versions of those pkgs for us to properly consume them.

Reply:
We can do this. I was going to bring this issue up in tomorrow's Extension meeting, too.

Reply:
Done! I'll update to 16.14 if iTwinUI React does.